registry/etcd.go

```diff
@@ -16,32 +16,28 @@ package registry
 
 import (
 	"path"
-	"time"
 
 	etcd "github.com/coreos/fleet/Godeps/_workspace/src/github.com/coreos/etcd/client"
 	"github.com/coreos/fleet/Godeps/_workspace/src/golang.org/x/net/context"
 )
 
 const DefaultKeyPrefix = "/_coreos.com/fleet/"
 
-func NewEtcdRegistry(kAPI etcd.KeysAPI, keyPrefix string, reqTimeout time.Duration) *EtcdRegistry {
+func NewEtcdRegistry(kAPI etcd.KeysAPI, keyPrefix string) *EtcdRegistry {
 	return &EtcdRegistry{
-		kAPI:       kAPI,
-		keyPrefix:  keyPrefix,
-		reqTimeout: reqTimeout,
+		kAPI:      kAPI,
+		keyPrefix: keyPrefix,
 	}
 }
 
 // EtcdRegistry fulfils the Registry interface and uses etcd as a backend
 type EtcdRegistry struct {
-	kAPI       etcd.KeysAPI
-	keyPrefix  string
-	reqTimeout time.Duration
+	kAPI      etcd.KeysAPI
+	keyPrefix string
 }
 
 func (r *EtcdRegistry) ctx() context.Context {
```

Comment:
This function doesn't add much value anymore - how about just inlining context.Background where it's called instead?